build.sbt

```diff
@@ -1,12 +1,20 @@
+import sbt._
+import Keys._
```

Comment:
A build.sbt implicitly has an `import sbt.Keys._` 

http://www.scala-sbt.org/0.13/docs/Basic-Def.html#Built-in+Keys